ui/src/app/components/buttons.tsx

```diff
@@ -239,13 +239,21 @@ const computeStyle = (
   };
 };
 
+interface ClickableProps {
```

Comment:
React Interactive allows for components to be used in the as attribute (not just html tags). Our testing for interactive buttons relies on looking for looking for the cursor: not allowed value on an inner element. This change prevents needless nesting of components.